.circleci/scripts/build_for_windows.sh

```diff
@@ -34,16 +34,77 @@ conda activate testenv
-python $DIR/remove_runnable_code.py beginner_source/dcgan_faces_tutorial.py beginner_source/dcgan_faces_tutorial.py || true
-python $DIR/remove_runnable_code.py intermediate_source/model_parallel_tutorial.py intermediate_source/model_parallel_tutorial.py || true
-python $DIR/remove_runnable_code.py intermediate_source/memory_format_tutorial.py intermediate_source/memory_format_tutorial.py || true
+export NUM_WORKERS=4
 
-make docs
+if [[ "${CIRCLE_JOB}" == *worker_* ]]; then
+  python $DIR/remove_runnable_code.py beginner_source/aws_distributed_training_tutorial.py beginner_source/aws_distributed_training_tutorial.py || true
+  python $DIR/remove_runnable_code.py intermediate_source/model_parallel_tutorial.py intermediate_source/model_parallel_tutorial.py || true
+  python $DIR/remove_runnable_code.py advanced_source/static_quantization_tutorial.py advanced_source/static_quantization_tutorial.py || true
+  export WORKER_ID=$(echo "${CIRCLE_JOB}" | tr -dc '0-9')
+  count=0
+  FILES_TO_RUN=()
+  for filename in $(find beginner_source/ -name '\*.py' -not -path '\*/data/\*'); do
+    if [ $(($count % $NUM_WORKERS)) != $WORKER_ID ]; then
+      echo "Removing runnable code from "$filename
+      python $DIR/remove_runnable_code.py $filename $filename
+    else
+      echo "Keeping "$filename
+      FILES_TO_RUN+=($(basename $filename .py))
+    fi
+    count=$((count+1))
+  done
+  for filename in $(find intermediate_source/ -name '\*.py' -not -path '\*/data/\*'); do
+    if [ $(($count % $NUM_WORKERS)) != $WORKER_ID ]; then
+      echo "Removing runnable code from "$filename
+      python $DIR/remove_runnable_code.py $filename $filename
+    else
+      echo "Keeping "$filename
+      FILES_TO_RUN+=($(basename $filename .py))
+    fi
+    count=$((count+1))
+  done
+  for filename in $(find advanced_source/ -name '\*.py' -not -path '\*/data/\*'); do
+    if [ $(($count % $NUM_WORKERS)) != $WORKER_ID ]; then
+      echo "Removing runnable code from "$filename
+      python $DIR/remove_runnable_code.py $filename $filename
+    else
+      echo "Keeping "$filename
+      FILES_TO_RUN+=($(basename $filename .py))
+    fi
+    count=$((count+1))
+   done
+   for filename in $(find recipes_source/ -name '\*.py' -not -path '\*/data/\*'); do
+    if [ $(($count % $NUM_WORKERS)) != $WORKER_ID ]; then
+      echo "Removing runnable code from "$filename
+      python $DIR/remove_runnable_code.py $filename $filename
+    else
+      echo "Keeping "$filename
+      FILES_TO_RUN+=($(basename $filename .py))
+    fi
+    count=$((count+1))
+   done
+   for filename in $(find prototype_source/ -name '\*.py' -not -path '\*/data/\*'); do
+    if [ $(($count % $NUM_WORKERS)) != $WORKER_ID ]; then
+      echo "Removing runnable code from "$filename
+      python $DIR/remove_runnable_code.py $filename $filename
+    else
+      echo "Keeping "$filename
+      FILES_TO_RUN+=($(basename $filename .py))
+    fi
+    count=$((count+1))
+  done
```

Comment:
Please use nested loops instead of repeating the same code
```suggestion
  for work_directory  in beginner_source intermediate_source advanced_source recipes_source prototype_source; do
    for filename in $(find $work_directory -name '\*.py' -not -path '\*/data/\*'); do
      if [ $(($count % $NUM_WORKERS)) != $WORKER_ID ]; then
        echo "Removing runnable code from "$filename
        python $DIR/remove_runnable_code.py $filename $filename
      else
        echo "Keeping "$filename
        FILES_TO_RUN+=($(basename $filename .py))
      fi
      count=$((count+1))
    done
  done
```

Reply:
done